pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -574,3 +585,20 @@ func applyMonitoringLabels(c *Cluster, serviceMonitor *monitoringv1.ServiceMonit
 		}
 	}
 }
+
+func (c *Cluster) getMGRsToSkipReconcile() (sets.String, error) {
+	listOpts := metav1.ListOptions{LabelSelector: fmt.Sprintf("%s=%s,%s", k8sutil.AppAttr, AppName, cephv1.SkipReconcileLabelKey)}
+	deployments, err := c.context.Clientset.AppsV1().Deployments(c.clusterInfo.Namespace).List(c.clusterInfo.Context, listOpts)
+
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to query MGRs to skip reconcile")
```

Comment:
```suggestion
		return nil, errors.Wrap(err, "failed to query mgrs to skip reconcile")
```

for consistency with other comments.